### DISCUSSION TITLE
Como agir quando pessoas faltam ao meetup?

### DISCUSSION THREAD

**Discussion by victormiguez** [frontendbr/forum#383] [Discussão]
Fala pessoal, tudo certo?

Gostaria de saber como vocês estão lidando com o problema de pessoas confirmarem a ida ao meetup e não comparecerem. Isso é comum na cidade de vocês? Em Campinas temos sempre uma fila de espera grande e uma média de falta de uns 10%.

Pelo que tenho visto, o mais comum é impedir que a pessoa compareça nas próximas edições.
O que acham que seja o mais justo e efetivo para ser feito nesses casos?

**Comment by vitor-mendrone-deel** [comment]
Invoco @matmarsiglio @LFeh

**Comment by mtmr0x** [comment]
![magic_cylinder](https://cloud.githubusercontent.com/assets/5414299/20885369/9e9b03e8-bad7-11e6-91ff-f7bb6abddf76.jpg)

Com essa carta eu devolvo pro @Mendrone a responsa de responder, mas mesmo assim responderei.

Tem duas formas de seguir com isso: Tu participar daquela seita (a seita que dói menos) ou banir das seguintes 2 ou 3 edições. Eu opto pelos dois sempre e ainda tomo uns atos para tentar diminuir o numero de ~~vacilos~~ faltas:

 - Mando email de ativação dois dias antes de fechar a lista e um dia antes de fechar a lista;
 - Procuro poder fechar a lista umas 6 horas antes do evento, o que me deixa mais tranquilo quanto a contra tempos dos participantes;
 - Como a média de faltas é 10%, eu libero para 10% a mais da capacidade do local, se lotar, eu e os amigos que vão sentamos no chão felizes pelo evento estar bombando.

Espero ter ajudado :)

att e 1 beijo

**Comment by eduardojmatos** [comment]
@victormiguez 10% é baixo se comparar por exemplo com o do HTML-SP, que às vezes chega a 25%... Eu acho bem ruim isso, principalmente quando não se tem uma desculpa por ter faltado. Tirar a vaga de outra pessoa aprender é uma coisa bem zoada!

**Comment by felipefialho** [comment]
@victormiguez Já tivemos uma média de mais de 25% de faltas no começo do Meetup. Hoje estabilizou, e infelizmente, acho difícil cair menos do que isso.

Mas o que o @raphaelfabeni e eu temos feito, é já colocar a lista com uma gordura, prevendo os faltantes. Isso evita cadeiras vazias e da espaço para mais gente ir. Tem funcionado bem.

**Comment by renatogalvones** [comment]
Reforçando a teoria dos 10%:

Essa regra dos 10% funciona até pra casamento cara. Experiência própria. =]

**Comment by victormiguez** [comment]
Boa, valeu pelas dicas!
Infelizmente isso rola mesmo, imprevistos acontecem, mas não são todos os casos :/

Já dá pra montar algumas regras para seguirmos \o/

**Comment by diogomoretti** [comment]
Aqui em Floripa eu organizo o @floripajs.

Usamos esse ano para fazer vários testes: já abrimos o RSVP só na semana do encontro, já passamos uma lista colhendo assinaturas para ter um frequencia/data do RSVP (tipo, mais da metade das pessoas que faltam, foi porque confirmaram bem antes) e também já anunciamos o meetup 1 semana antes do encontro.

A nossa taxa de ausência é relativamente alta (20 ~ 30), e muitas vezes chega a 40~50. Aqui em Floripa é relativamente pequeno e a principio não queremos fazer punições.

Vejo que a consciência do pessoal tá mudando e ano que vem o RSVP só abrirá 3~5 antes do encontro, pois vejo que essa é a forma que deixa a menor taxa de ausência.

[]'s

**Comment by notnotgabriel** [comment]
@diogomoretti como o de Campinas é pequeno também, no principio a gente não procurava fazer essas punições mesmo. Agora percebemos que tem ficado algumas pessoas na lista de espera, então queremos ser mais justos nesse ponto.
Seria legal a opinião de uma pessoa que já foi punida (sei que é díficil) hauahuha

Valeu pelas respostas galera!

**Comment by cristofersousa** [comment]
Aqui em São Carlos/Araraquara eu organizo o [@opensanca ](https://github.com/opensanca/), adotamos a politica de gancho de 3 eventos.

**Comment by yanmagale** [comment]
Cara, esse fórum é o melhor de todos do Brasil, só tópicos relevantes :+1: 

Então, isso é muito complicado. Lidar com o no-show é algo que faz parte desse mundo dos organizadores de eventos e meetups. Infelizmente a cultura das pessoas é de deixar tudo para a última hora. O problema é que nem tudo é de graça em um evento: custos com coffe-break, deslocamento, o tempo e energia que foram gastos para promover o evento, custos com o espaço e etc, tudo isso são elementos que as vezes não temos noção quando simplesmente marcamos o RSVP como não a 30 minutos do evento, ou nem sequer fazemos isso. Além do fato da pessoa não ir, acabar ocupando o espaço de uma outra pessoa que poderia ter ido em seu lugar. Acho que a primeira parte da minha resposta cabe a uma reflexão de todos nós como comunidade, para diminuirmos essa política de deixar tudo para a última hora e termos uma presença mais efetivia nos meetups.

Aqui em BH, organizo o GDGBH e o FEMUG-MG. No GDG, não adotamos nenhuma política de banimento.  No Femug, fechamos o RSVP com alguns dias de antencedência, caso a pessoa falte no dia, e apresente uma justificativa plausível (claro, imprevistos podem acontecer), ela é marcada na nossa lista com um Warning e enviamos um e-mail a ela. Caso a pessoa falte e não nos comunique, é feito um banimento de 3 encontros e a pessoa recebe um e-mail sobre isto.

**Comment by raphaelfabeni** [comment]
Apenas completando o que o @LFeh disse: no meetup CSS aqui em SP fazemos algumas coisas:

* Tentamos abrir/divulgar o evento com pelo menos 10 dias de antecedência pra galera poder se organizar.
* Fechamos a lista 24h antes do evento.
* Durante o dia da edição, até determinado horário, conforme as pessoas vão desistindo, vamos tentando manualmente preencher esses espaços.
* Dentre essas pessoas que vão desistindo, algumas poucas avisam. As que não avisam (seja não aparecendo ou simplesmente saindo da lista), vão direto pra lista de _ban_ (como já havíamos fechado a lista no dia anterior, conseguimos saber quem apareceu ou não).
* Como dito, sempre colocamos em torno de 15% a mais de pessoas que hoje é a nossa média de ausências.
